index.html

```diff
@@ -1692,8 +1688,93 @@ <h5>Implementations</h5>
         </aside>
         <img src="img/room_perspective.png" />
 
+        <p>To establish a tunnel, one of the peers (the "source") calls the RPC <em>tunnel.connect</em> on the portal peer, specifying the "target" peer's ID:</p>
+        <figure class="request-response">
+            <div class="request">
+                <div class="header">
+                    <span class="key">Request number</span><span class="value">1</span>
+                    <span class="key">Body type</span><span class="value">JSON</span>
+                    <span class="key">Stream</span><span class="value">Yes</span>
+                    <span class="key">End/err</span><span class="value">No</span>
+                </div>
+                <pre><code>{
+  "name": ["tunnel", "connect"],
+  "type": "duplex",
+  "args": [
+    {
+      "portal":"@uMYDVPuEKftL4SzpRGVyQxLdyPkOiX7njit7+qT/7IQ=.ed25519",
```

Comment:
Why pass the portal as argument? Isn't that always the recipient of the message? If there is a reason for this, please explain the reason in the prose before the example and explain how the recipient should handle the situation when they are not the specified portal. Otherwise remove.

Reply:
This PR is not a proposal for a new protocol, it's a description of an existing protocol. I understand suggestions for improvements on the tunnel protocol, but this PR is not the space for improving the tunnel protocol. It's descriptive, not prescriptive.